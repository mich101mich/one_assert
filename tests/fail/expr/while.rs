fn main() {
    one_assert::assert!(while false {});
}
