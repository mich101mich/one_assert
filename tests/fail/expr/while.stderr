error: Expected a boolean expression, found a while loop
 --> tests/fail/expr/while.rs:2:25
  |
2 |     one_assert::assert!(while false {});
  |                         ^^^^^^^^^^^^^^
